### PR TITLE
Crud operation on category list using Vue Js complete

### PR DESCRIPTION
**Pull request by swarup-ranium**

### CODE REVIEW

File: app/Http/Controllers/TaskCategoriesController.php [hunk 1]
```diff
@@ -21,8 +21,8 @@ class TaskCategoriesController extends Controller
     public function index(Request $request)
     {
         $categories = $request->user()->taskCategories;
-
-        return view('task-category.index', compact('categories'));
+        return $categories->toArray();
```
Comment (abbasali) [inline app/Http/Controllers/TaskCategoriesController.php]: Again, use API Resource to return json.

**Review by abbasali — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/TaskCategoriesController.php (hunk 1 above)
Comment (swarup-ranium) [reply app/Http/Controllers/TaskCategoriesController.php]: ok

File: app/Http/Controllers/TaskCategoriesController.php [hunk 2]
```diff
@@ -22,7 +22,10 @@ public function index(Request $request)
     {
         $categories = $request->user()->taskCategories;
 
-        return view('task-category.index', compact('categories'));
+        TaskCategoryResource::withoutWrapping();
+        return new TaskCategoryResource($categories);
```
Comment (abbasali) [inline app/Http/Controllers/TaskCategoriesController.php]: For returning a collection you need to use `return TaskCategoryResource::collection($categories)`

**Review by abbasali — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/TaskCategoriesController.php (hunk 2 above)
Comment (swarup-ranium) [reply app/Http/Controllers/TaskCategoriesController.php]: ok sir.

**Review by amit-ranium — COMMENTED** [review]
Looks fine. Please remove old commented code and debug statements. Also configure your editor to cleanup trailing whitespace automatically on save.

**Review by amit-ranium — APPROVED** [review]

**Pull request merged**